src/index.js

```diff
@@ -1,7 +1,7 @@
 grapesjs.plugins.add('gjs-blocks-basic', (editor, opts) => {
   let c = opts || {};
   let config = editor.getConfig();
-  let pfx = config.stylePrefix;
+  let stylePrefix = 'blocks-basic'
```

Comment:
Can you put `let pfx = config.stylePrefix;` back? 
I'd like the `stylePrefix` to be configurable by users. 
As a default prefix, I'd rather prefer to use `gjs-` instead of `blocks-basic`